docs/serving/samples/cloudevents/cloudevents-go/Dockerfile

```diff
@@ -6,17 +6,17 @@ FROM golang:1.13 as builder
 # Create and change to the app directory.
 WORKDIR /app
 
-# Retrieve application dependencies using go modules.
-# Allows container builds to reuse downloaded dependencies.
-COPY go.* ./
-RUN go mod download
-
 # Copy local code to the container image.
 COPY . ./
 
-# Build the binary.
-# -mod=readonly ensures immutable go.mod and go.sum in container builds.
-RUN CGO_ENABLED=0 GOOS=linux go build -mod=readonly -v -o server
+# Ensure Go modules are on and create the go.mod and go.sum files.
+# Also builds the binary
+ENV GO111MODULE=on
+RUN go mod init cloudevents.go \
+    && CGO_ENABLED=0 GOOS=linux go build -v -o server
+
+# Allows the container builds to reuse downloaded dependencies.
```

Comment:
This probably doesn't help much because the `COPY` on line 10 is going to produce a new image